apicast/src/policy/load_configuration.lua

```diff
@@ -0,0 +1,33 @@
+local setmetatable = setmetatable
+
+local _M, mt = require('policy').new()
```

Comment:
We should probably call `new()` with a name. Otherwise, it will use the default one.

Reply:
Good point!

Reply:
All the policies are now initialized with names :+1: